addons/stock/models/stock_orderpoint.py

```diff
@@ -309,8 +309,10 @@ def _get_orderpoint_action(self):
         to_refill = defaultdict(float)
         all_product_ids = []
         all_warehouse_ids = []
+        ConfigParameter = self.env['ir.config_parameter'].sudo()
+        forecast_forecast_interval = ConfigParameter.get_param("stock.forecast_interval", "3")
         # Take 3 months since it's the max for the forecast report
-        to_date = add(fields.date.today(), months=3)
+        to_date = add(fields.date.today(), months=int(forecast_forecast_interval))
```

Comment:
The comment above this line also has to be updated, if the default interval of 3 months can be configured ;)